ansible/configs/osp-migration/infra.yml

```diff
@@ -81,6 +81,23 @@
         create_objects_user: "{{ guid }}"
         create_objects_password: "{{ osp_migration_api_pass }}"
       when: use_openstack_centralized_router | default(False) == False
+      
+    - name: Gather OSP network external network
+      os_networks_facts:
+        name: "{{ osp_external_network | default('external') }}"
+        filters:
+          router:external: true
+      register: r_osp_n_facts
+
+    - name: Fail is there is not an external network found
+      assert:
+        that:
+          - (r_osp_n_facts.ansible_facts.openstack_networks | count) > 0
+        msg: "No OSP external network found"
+
+    - set_fact:
+        external_network:
```

Comment:
Please include a name for every task.

Also, is it difficult to change this variable name to something like `osp_external_network`?